### ISSUE TITLE
1ATOM deposits disappear

### ISSUE THREAD

**Issue by IgorGer55** [persistenceOne/pStake-bugBounty#16]
**1ATOM deposits disappear**
When transferring 1 ATOM from the KEPLR wallet to the platform, these deposits are lost. They are not in the history of transactions or on the balance sheet.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to https://alpha.pstake.finance/
2. Connect Meta Mask wallet in Ropsten network
3. Connect Keplr wallet in pSTAKE alpha testnet network
4. Click on 'wrap function'
5. choose 1 ATOM
6. Click on 'wrap button'
7. Choose a Keplr your ATOM via Either Method
8. Approve transaction in Keplr wallet
9. We receive a push notification on the pstake.finance platform that the transaction is successful
10. The balance did not change, and this deposit will not appear in the transaction history
11. 1 ATOM for the user is lost

**Expected behaviour**
The client will consider the service a scam, where funds are lost

**Screenshots**
![alt text](https://github.com/IgorGer55/Pickchers/blob/main/%D0%BE%D0%BD%D0%BE.jpg?raw=true)

**Desktop**
 - OS: Windows 8.1
 - Browser - Google chrome
 - Version 91.0.4472.114

**Ethereum address**
0xCf7FB3AdC0A9A4E6F7Caff2ca8C365325Fd19277

**Criticality Assessment**
Please pick one:
- [x] **High**:	An issue that might cause immediate loss of the funds or permanent/severe damage of the protocol state
- [ ] **Medium**:	An issue that might theoretically cause minimal loss of funds damage the protocol state or cause severe user dissatisfaction
- [ ] **Low**:	An issue that might cause user dissatisfaction or minimal failure of the application

**Checklist**
- [x] The reported issue is in the [scope](https://docs.pstake.finance/Program_Scope/) of the pStake BugBounty program.
- [x] This issue has not been reported before.
- [x] The ethereum address filled in is valid.

**Comment by napelvs** [comment]
I could partially reproduce this. There is a minimum deposit amount of 2 ATOMs which is not mentioned in the UI. However, the funds are not lost. The ATOM token from the Keplr wallet hasn't been withdrawn.

**Comment by IgorGer55** [comment]
> Я мог частично воспроизвести это. Существует минимальная сумма депозита в 2 ATOM, которая не упоминается в пользовательском интерфейсе. Однако средства не потеряны. Токен ATOM из кошелька Keplr не был выведен.

This is not true. The keplr balance decreases. I sent when my wallet had 8 ATOMs. Then it became 7.

**Comment by kombos** [comment]
any wrap txn with amount less than 2 ATOMs gets reversed after some time. your funds are not lost. They are not showing in the pstake txns history since it got reversed.
